addon/styles/_frost-tab.scss

```diff
@@ -1,9 +1,8 @@
-$frost-tab-active-border: $frost-color-blue-1;
+$frost-tab-active-border: $frost-color-grey-2;
```

Comment:
based on discussion with @baconC, the line was to be changed to grey-2 for any situation